### ISSUE TITLE
Update urls on the main page

### ISSUE THREAD

**Issue by dsych** [humphd/next#141]
I was trying to show this project to some colleagues of mine and it seems like the urls are broken as is. When I manually update the urls, everything works fine accept for the terminal. See the details below:

1. The main page now contains broken urls. They are pointing to '/some-path' when in reality content is hosted at '/next/some-path'.
2. /terminal is trying to fetch resources from '/next' instead of '/next/terminal' e.g. css files etc.

**Comment by humphd** [comment]
Hey, nice to hear from you!  If you have time to fix this, and want to send a PR, I'd take it.  I probably won't keep iterating on this though, since I've split the work off into another experiment I published here:

https://github.com/humphd/browser-shell
https://humphd.github.io/browser-shell/

**Comment by dsych** [comment]
Cool, I will take a look at. Is https://github.com/humphd/browser-shell being actively developed?

**Comment by humphd** [comment]
I'm not sure what I'll do with it.  I mostly wanted to clean up what we'd started here, and get a demo + writeup on the web.  I might keep iterating on it, or I might leave it as is.  Not sure.  Do you have ideas for it?

I will almost certainly do something with the ideas in it, whether I continue with that repo or not.

**Comment by dsych** [comment]
One thing I found while playing around with your demo: 
Our old project https://humphd.github.io/next/editor/ seems to be hooking into your new project https://humphd.github.io/browser-shell/ .
Try creating some files in the terminal and the refreshing the editor. Files will show up. The same is true the other way around for editing file through the editor.

**Comment by humphd** [comment]
They both use filer, so they would be mounting the same db via the https://humphd.github.io/ origin.

**Comment by dsych** [comment]
Yeah, I know. Just thought that it was. This also means that you can add
our old editor to your demo and it should work out of the box.

**Comment by humphd** [comment]
Ah.  I didn't include the editor because it was a bit messy how we included it.  I still love the idea of layering on "apps" like this though, just didn't have time to hack one into my demo.  Cool that it "just works" though!

**Comment by dsych** [comment]
In terms of ideas, I think it would be super neat to be able to install
packages into the VM. Maybe write our own implementation of a package
manager that will download and install pre-complied binaries.
